### PR TITLE
Added Rooms List View and Common Header Helper

### PR DESCRIPTION
**Pull request by wahibkapdi**

### CODE REVIEW

**Review by Naviunni — CHANGES_REQUESTED** [review]
Please add cucumber features as well

**Comment by wahibkapdi** [comment]
> Please add cucumber features as well

Yeah working on it.

File: db/parse_rooms_to_csv.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                    if type == 1:
+                        lecture_hall = True
+                    elif type == 2:
+                        learning_studio = True
+                    elif type == 3:
+                        lab = True
+            campus = "CS"
+
+            if entry['campus'] == 2:
+                campus = "GV"
```
Comment (Naviunni) [inline db/parse_rooms_to_csv.py:35]: Isn't it better to make a map for these kind of mappings rather than if else statements?

**Review by Naviunni — APPROVED** [review]

File: db/parse_rooms_to_csv.py (hunk 1 above)
Comment (wahibkapdi) [reply db/parse_rooms_to_csv.py:35]: This is not meant for the final application it is a one time thing to convert our json to csv.

**Pull request merged**